### PR TITLE
Zint package

### PR DESCRIPTION
**Pull request by et4te**
Package for the zint barcode generation library written in C.

### CODE REVIEW

File: pkgs/applications/editors/emacs-modes/slime/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  buildInputs = [emacs];
+  installPhase = ''
+    rm -rf CVS
+    mkdir -p $out/share/emacs/site-lisp
+    cp -r . $out/share/emacs/site-lisp
+  '';
+  meta = {
+    homepage = "https://common-lisp.net/project/slime/";
+    description = "The Superior Lisp Interaction Mode for Emacs";
+    license = "GPL";
```
Comment (pSub) [inline pkgs/applications/editors/emacs-modes/slime/default.nix]: Use a license from `lib.licenses`.

File: pkgs/applications/editors/emacs-modes/slime/default.nix [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    sha256 = "06ncqxzidmis6d7xsyi5pamg4pvifmc8l854xaa847rhagsvw7ax";
+  };
+  buildInputs = [emacs];
+  installPhase = ''
+    rm -rf CVS
+    mkdir -p $out/share/emacs/site-lisp
+    cp -r . $out/share/emacs/site-lisp
+  '';
+  meta = {
+    homepage = "https://common-lisp.net/project/slime/";
```
Comment (pSub) [inline pkgs/applications/editors/emacs-modes/slime/default.nix]: Nitpick: You don't need quotes around URLs.

File: pkgs/applications/misc/zint/default.nix [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchgit, libpng, zlib, cmake, qt4 }:
+
+let
+  version = "2.4.2";
+in
+stdenv.mkDerivation {
+  name = "zint-${version}";
+
+  src = fetchgit {
```
Comment (pSub) [inline pkgs/applications/misc/zint/default.nix]: Please use fetchFromGitHub.

File: pkgs/applications/misc/zint/default.nix [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  };
+
+  buildInputs = [ cmake stdenv libpng zlib qt4 ];
+
+  configurePhase = ''
+    cmake -DCMAKE_INSTALL_PREFIX=$out
+  '';
+
+  meta = {
+    homepage = "https://zint.github.io";
```
Comment (pSub) [inline pkgs/applications/misc/zint/default.nix]: You don't need quotes around URLs.

**Comment by pSub** [comment]
Please rebase against master.

**Comment by zimbatm** [comment]
Closing because this work has been abandoned. I you want to keep working on this feel free to re-open the PR.

**Pull request closed without merging**